README.md

```diff
@@ -324,6 +324,50 @@ See the Apollo Client documentation for [Authentication Params Over Websocket](h
 
 See the Keycloak Documentation for the [Keycloak JavaScript Adapter](https://www.keycloak.org/docs/latest/securing_apps/index.html#_javascript_adapter)
 
+## Usage with Apollo Federation
+As your data graph grows, however, it can become inefficient or even difficult to represent the graph with a single, monolithic GraphQL server. To remedy this, you can divide your data graph's implementation across multiple composable services with **Apollo Federation**.
```

Comment:
To me, this language is a little bit like something you might read in a blog post rather than something you would write in documentation. I would really simplify this down to something like 

```
`keycloak-connect-graphql` can be used with [Apollo Federation]((https://www.apollographql.com/docs/apollo-server/federation/introduction/).
```

Reply:
Oh ya, sorry about that. Thank you @darahayes .